spec/index.bs

```diff
@@ -1855,6 +1857,11 @@ A <dfn dfn>supported limits</dfn> object has a value for every limit defined by
     <tr class=row-continuation><td colspan=4>
         The maximum value for the arguments of
         {{GPUComputePassEncoder/dispatchWorkgroups(workgroupCountX, workgroupCountY, workgroupCountZ)}}.
+    <tr><td><dfn>maxClipDistancesSize</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>0
+        The maximum value for the array size of [=builtin/clip_distances=] in a vertex stage output.
+        When the {{GPUFeatureName/"clip-distances"}} feature is enabled the default value of
+        `maxClipDistancesSize` is 8, otherwise it is always 0.
```

Comment:
I'd like the editors to have a discussion about whether we do this, or make the default 8 but you just can't use them unless the feature is available. (Then we'd bring our recommendation to the WG.) I feel like I had a plan for this but I'm not sure I ever fully formed one.

Both have their pros and cons:
- If the default is 8, we don't have to add complicated rules like "if you ask for the limit but not the feature, the device request fails; if you ask for the feature but not the limit, you get 8"
- If the default is 0, applications can just look at the limit and ignore whether the feature is available

Reply:
I think we generally would prefer to add limits for things that may change over time, but it's certainly an option. We could even do that and add the limit later.

Reply:
Discussed how a new limit would work with Brandon yesterday, started drafting in #4613

Reply:
> I think we generally would prefer to add limits for things that may change in the future, but it's certainly an option. We could even omit the limit and then add it later.

I also think always choosing 8 will make everything simpler so now I prefer to do so in this PR. What do you think?

Reply:
IMO let's go with static 8 in this PR. If we do decide we want to add the limit, we have the option of waiting for that before Chrome ships the feature, if we want to avoid minor ecosystem fragmentation.